requirements.txt

```diff
@@ -5,6 +5,7 @@ djangorestframework>=3.11,<3.12
 django-filter>=2.2,<2.3
 django-reversion>=3.0,<3.1
 django-cors-headers>=3.3,<3.4
+cerberus>1.3,<1.4
```

Comment:
Why is this >1.3,<1.4 and not >=1.3,<1.4?

Reply:
eh just because the current version is greater than 1.3 already (1.3.2 I think). I'm fine changing it to >= though...

Reply:
Hmm yeah I guess setting a >= would be my preference, it seems good to have a lower inclusive bound. Could be >=1.3.2 if that's preferable.